Formula/tfel.rb

```diff
@@ -0,0 +1,69 @@
+class Tfel < Formula
+  desc "Main repository of TFEL/MFront project"
+  homepage "https://thelfer.github.io/tfel/web/index.html"
+  license "GPL-1.0-or-later"
+  head "https://github.com/thelfer/tfel.git", using: :git, branch: "master"
+  stable do
+    url "https://github.com/thelfer/tfel/archive/refs/tags/TFEL-4.0.tar.gz"
+    sha256 "ed3fb2f59c6b8c9896606ef92276f81942433dd5f60d8130ba07c3af80b039e2"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :required
```

Comment:
Why do we need `gcc`?

Reply:
Hi @carlocab, thanks for your comments.
`gcc` is needed to use TFEL/MFront under linux and `clang` is needed under MacOS. Is there a generic "C++ compiler" package?

Reply:
Homebrew does not work on Linux without `gcc`, so there is no need to say `depends_on "gcc"` if you are okay with a generic GCC.